src/lib/pwa/sync-service.ts

```
@@ -0,0 +1,320 @@
+  }
+  
+  private async syncEntry(entry: SyncEntry): Promise<void> {
+    const supabase = createClient();
+    
+    switch (entry.type) {
+      case 'meal_log':
+        await this.syncMealLog(supabase, entry.data as MealLogData);
+        break;
+        
+      case 'photo_upload':
+        const signedUrl = await this.syncPhotoUpload(supabase, entry.data as PhotoUploadData);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)